FreeRTOS/FreeRTOS-Kernel
# Fix: Remove the newly (mistakenly) added GCC/RX200 port

Description
-----------
The newly (mistakenly) added GCC/RX200 port has to be removed as I reported in the Related Issue.

Test Steps
-----------
Test isn't necessary. There are no demos which use this port.

Related Issue
-----------
[BUG] The newly added GCC/RX200 port is wrong.
https://github.com/FreeRTOS/FreeRTOS-Kernel/issues/139

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
